src/stormpath.social-login.linkedin.js

```diff
@@ -0,0 +1,43 @@
+(function() {
+  'use strict';
+
+  function loadLinkedInSDK($spJsLoader, clientId) {
+    var innerHTML = 'api_key: ' + clientId + '\n' +
```

Comment:
This doesn't say anything about what `api_key` etc comes from. I'd have a separate method:

```js
// Comment describing where to reference the LinkedIn config format.
function buildLinkedInJsConfig(clientId) {
  return 'api_key: ' + clientId + '\nauthorize: true';
}
```